build/install-dotnet.ps1

```diff
@@ -7,4 +7,6 @@
 
 &([scriptblock]::Create((Invoke-WebRequest -UseBasicParsing 'https://dot.net/v1/dotnet-install.ps1'))) -Channel 7.0
```

Comment:
Should `7.0` be removed since we drop `net7` target

Reply:
Tracked in https://github.com/microsoft/FeatureManagement-Dotnet/issues/517